### PR TITLE
doc(intercom): dbt source README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
I have edited this readme to follow the Fivetran docs style guide.

@fivetran-joemarkiewicz See questions line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +45,7 @@ vars:
     contact_history_pass_through_columns: [super_cool_contact_field]
 ```
 
-Additionally, this package includes company, contact, and conversation tag mapping tables as well as a company contact mapping table. We understand not every Intercom customer utilizes these tables. As such, if you do not use these tables you may add the below configuration (removing any configs for tables you use) to your `dbt_project.yml` file. By default these variables are set to `True`:
+Additionally, this package includes Intercom's `company`, `contact`, `company contact`, and `conversation tag` mapping tables. If you do not use these tables, add the configuration below to your `dbt_project.yml`. By default, these variables are set to `True`:
```
Comment (fivetran-chloe) [inline README.md:48]: Please double-check the table names

File: README.md [hunk 2]
```diff
@@ -34,8 +33,7 @@ vars:
     intercom_schema: your_schema_name
 ```
 
-This package includes all source columns defined in the macros folder. To add additional columns to this package, do so using our pass-through column variables. This is extremely useful if you'd like to include custom fields to the package.
-
+This package includes all source columns defined in the macros folder. If you want to include custom fields in this package, you can add more columns using our pass-through column variables.
```
Comment (fivetran-chloe) [inline README.md:36]: Please double-check this content - I reworded for clarity, but I want to be sure I didn't change your original meaning

File: README.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: Thanks! I updated the table names.

File: README.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply README.md:36]: original meaning persists and is much clearer. Thank you!

**Pull request merged**